### DISCUSSION TITLE
How to make first split overrule destination account of other split?

### DISCUSSION THREAD

**Discussion by matejdro** [firefly-iii/firefly-iii#4918] [Firefly III]
When creating a split transaction, I get this notice:

![image](https://user-images.githubusercontent.com/507922/125568091-699c34e1-6e96-42fc-8f03-de63b8eda371.png)

I assume this means that I could enter destination account on only the first split and it would automatically be filled on the subsequent splits? But that does not happen, they remain blank after saving the transaction.

What exactly does this notice mean? Can I tell Firefly to use one destination on all splits?

Thanks.

**Comment by JC5** [comment]
If you have found a bug, could you explain how to reproduce it?

**Reply by matejdro** [reply]
I'm not sure if it is a bug. I'm asking because I do not know what the desired behavior is.

This is what I do:

1. Create a transaction 
2. Add a split
3. Fill in first split (with Destination account)
4. Fill in second split (but leave Destination account empty)
5. Save transaction
6. Second split will have default cash account set as a destination (presumably it should have destination from the first split?)

This is with 5.5.12

**Reply by JC5** [reply]
Thanks for letting me know. Is this in the new layout or the old one? The behaviour is not what it's supposed to be 😁

**Reply by matejdro** [reply]
It's the new layout.

**Reply by JC5** [reply]
Wait no this is not a bug this is by design. If you dont fill it in it falls back to cash. The message can be removed.

**Reply by matejdro** [reply]
Hm, can I turn this into a feature request? Maybe an option that would perform this sort of filling?

90% of my split use is splitting single bill to single recipient into multiple transactions, so it would be useful to not have to specify the destination every single time.

**Reply by JC5** [reply]
I'll see what I can do, but don't hold your breath. The split function is specifically to set different destinations.
